emat/workbench/em_framework/util.py

```diff
@@ -213,9 +213,12 @@ def combine(*args):
     for entry in args[1::]:
         overlap = set(experiment.keys()).intersection(set(entry.keys()))
         if overlap:
-            raise EMAError(
-                'parameters exist in {} and {}, overlap is {}'.format(
-                    experiment, entry, overlap))
+            print('Warning: Please check the parameters overlap among experiment and entry')
```

Comment:
This should use format strings and use python logging functionality to print the error

Reply:
I will make this change and convert it to a warning message to the user.